extensions/adobe/experience/decisioning/option-detail.schema.json

```diff
@@ -104,6 +104,7 @@
           "description": "Option selection type. The selection type of which the item was derived from."
         },
         "xdm:trackingToken": {
+          "meta:status": "deprecated",
```

Comment:
@kenadobe , just FYI, marking a field `deprecated` merely means that it will not show up in the UI. But someone can still ingest data in this field.